doc/sailfish-webview-webengine.qdoc

```diff
@@ -0,0 +1,326 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC
+**
+****************************************************************************/
+
+/*!
+    \qmltype WebEngine
+    \inherits Item
+    \inqmlmodule Sailfish.WebEngine
+    \ingroup sailfishos-webview
+    \brief Provides access to the Web engine context.
```

Comment:
maybe remove capitalisation from Web

Reply:
Removed (even though it's a bit of a struggle accepting that web's not considered a proper noun any more; old habits).